packages/blocks/src/page-block/edgeless/components/note-cut/cut-button.ts

```diff
@@ -139,7 +139,7 @@ export class NoteScissorsButton extends WithDisposable(LitElement) {
         position: absolute;
         left: 0;
         top: 0;
-        z-index: 1;
+        z-index: calc(var(--affine-z-index-popover, 0) + 2);
```

Comment:
Is it possible to define this `calc` result as a new CSS variable like `--affine-z-index-edgeless-scissor`?